Makefile

```diff
@@ -91,17 +91,18 @@ setup-gotools:
 	go get -u golang.org/x/tools/cmd/cover
 	go get -u golang.org/x/tools/cmd/vet
 
-test: test-style test-unit
-
 test-style:
 	${DEV_ENV_CMD} sh -c '${GOFMT} ${GO_FILES} ${GO_PACKAGES} && go vet $(repo_path) $(GO_PACKAGES_REPO_PATH)'
 	@for i in $(addsuffix /...,$(GO_PACKAGES)); do \
 		${DEV_ENV_CMD} golint $$i; \
 	done
 
-test-unit:
+test: test-style
 	${DEV_ENV_PREFIX_CGO_ENABLED} ${DEV_ENV_IMAGE} sh -c '${GOTEST} $$(glide nv)'
 
+test-cover: test-style
+	${DEV_ENV_PREFIX_CGO_ENABLED} ${DEV_ENV_IMAGE} test-cover.sh
```

Comment:
It's inside the [docker-go-dev container](https://github.com/deis/docker-go-dev/blob/master/rootfs/usr/local/bin/test-cover.sh)